src/Stryker.Core/Stryker.Core/Compiling/RollbackProcess.cs

```diff
@@ -30,7 +31,7 @@ public RollbackProcess()
             _rollbackedIds = new List<int>();
         }
 
-        public RollbackProcessResult Start(CSharpCompilation compiler, ImmutableArray<Diagnostic> diagnostics,
+        public RollbackProcessResult Start(CSharpCompilation compiler, ImmutableArray<Diagnostic> diagnostics, bool lastAttempt,
```

Comment:
Is there any reason you have `lastAttempt` and `devMode` in a different order below (Line 157)? I'm not sure of Stryker's coding standards, but this would bug me.

Reply:
fixed